docs/auth/basic.rst

```diff
@@ -2,9 +2,13 @@ HTTP Basic Authentication Policy
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 To adopt basic HTTP authentication, you can use build-in authentication policy
-``BasicAuthAuthenticationPolicy``
+``BasicAuthAuthenticationPolicy``.
 
-Use it something like::
+Please check the
+`BasicAuthAuthenticationPolicy <http://docs.pylonsproject.org/projects/pyramid/en/latest/api/authentication.html#pyramid.authentication.BasicAuthAuthenticationPolicy>`_
+manual entry for more details.
+
+Use it like this::
 
```

Comment:
Let's condense the above and correct the grammar of the original sentence in one swoop. As an aside, you do not need to hard wrap narrative docs.

```
To adopt basic HTTP authentication, you can use Pyramid's built-in authentication policy, `BasicAuthAuthenticationPolicy <http://docs.pylonsproject.org/projects/pyramid/en/latest/api/authentication.html#pyramid.authentication.BasicAuthAuthenticationPolicy>`_.
```